packages/@coorpacademy-app-review/src/types/common.ts

```diff
@@ -1,10 +1,18 @@
 import {Slide} from './slides';
 
+// TODO use type from components
+type TemplateContextValues = {
```

Comment:
We could move it to ./variables/template.tsx

Reply:
directement dans template/app-review/template-context meme, mais il faut exporter les types depuis components pour pouvoir les importer ensuite;
on affinera ça plus tard